File: SGK9/web/js/Acta.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

function Acta(nacta,informe,ubicacion,imputados,detalles,testigoact,imputadofirma,oficialdecomiso) {
    this.Acta(nacta,informe,ubicacion,imputados,detalles,testigoact,imputadofirma,oficialdecomiso);
  }
  
  Acta.prototype={
  	nacta: "",
	informe: "",
        ubicacion: new Sitio(),
        imputados:[],
	detalles: "",
        testigoact: new Testigo(),
        imputadofirma: new Imputado(),
        oficialdecomiso: new OficialAc(),
	Acta: function(nacta,informe,ubicacion,imputados,detalles,testigoact,imputadofirma,oficialdecomiso){
		this.nacta=nacta;
		this.informe=informe;
                this.ubicacion=ubicacion;
                this.imputados=imputados;
		this.detalles=detalles;
                this.testigoact=testigoact;
                this.imputadofirma=imputadofirma;
		this.oficialdecomiso=oficialdecomiso;
	},
	toString:function(){
	  return this.nacta;
	}
  };
  
  Acta.from= function(plain){
    var acta = new Acta(plain.nacta,plain.informe,plain.ubicacion,plain.imputados,plain.detalles,plain.testigoact,plain.imputadofirma,plain.oficialdecomiso);
	return acta;
  };
  
    Acta.to= function(acta){
    return {
        _class : 'Acta',
        nacta : acta.nacta,
        informe : acta.informe,
        ubicacion : acta.ubicacion,
        imputados : acta.imputados,
	detalles : acta.detalles,
        testigoact : acta.testigoact,
        imputadofirma : acta.imputadofirma,
	oficialdecomiso : acta.oficialdecomiso
    };	
  };

File: SGK9/src/java/model/Acta.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.io.Serializable;
import java.util.ArrayList;

/**
 *
 * @author Leo
 */
public class Acta implements Serializable, Jsonable{
    
    String nacta;
    String informe;
    Sitio ubicacion;
    ArrayList<Imputado> imputados;
    String detalles;
    Testigo testigoact;
    Imputado imputadofirma;
    OficialAc oficialdecomiso;
    

    public String getActa() {
        return nacta;
    }

    public void setActa(String acta) {
        this.nacta = acta;
    }

    public String getInforme() {
        return informe;
    }

    public void setInforme(String informe) {
        this.informe = informe;
    }

    public String getDetalles() {
        return detalles;
    }

    public void setDetalles(String Detalles) {
        this.detalles = Detalles;
    }

    public String getNacta() {
        return nacta;
    }

    public void setNacta(String nacta) {
        this.nacta = nacta;
    }

    public Sitio getUbicacion() {
        return ubicacion;
    }

    public void setUbicacion(Sitio ubicacion) {
        this.ubicacion = ubicacion;
    }

    public ArrayList<Imputado> getImputados() {
        return imputados;
    }

    public void setImputados(ArrayList<Imputado> imputados) {
        this.imputados = imputados;
    }

    public Testigo getTestigoact() {
        return testigoact;
    }

    public void setTestigoact(Testigo testigoact) {
        this.testigoact = testigoact;
    }

    public Imputado getImputadofirma() {
        return imputadofirma;
    }

    public void setImputadofirma(Imputado imputadofirma) {
        this.imputadofirma = imputadofirma;
    }

    public OficialAc getOficialdecomiso() {
        return oficialdecomiso;
    }

    public void setOficialdecomiso(OficialAc oficialdecomiso) {
        this.oficialdecomiso = oficialdecomiso;
    }
    
    
    
}
